OWNERS

```diff
@@ -1,19 +1,13 @@
 approvers:
-  - leodido
-  - fntlnz
-  - mfdii
-  - kris-nova
   - leogr
```

Comment:
```suggestion
  - leogr
  - Issif
  - Andreagit97
  - terylt
  - maxgio92
  - araujof
```

I believe we have already reached a good number of maintainers here. Note that this is a special repository, and it won't probably need too much maintenance. However, any idea to improve the repository is always welcome!

PS if other maintainers still want to join, please propose yourself! (even after this PR gets merged)